compactcert/signer.go

```diff
@@ -183,7 +184,8 @@ func (ccw *Worker) signBlock(hdr bookkeeping.BlockHeader) {
 			ccw.log.Warnf("ccw.signBlock(%d): GenerateStateProofMessage: %v", hdr.Round, err)
 			continue
 		}
-		sig, err := key.StateProofSecrets.SignBytes(commitment)
+		tmp := commitment.Hash()
```

Comment:
We can move this code + the call to GenerateStateProofMessage to outside of the loop.

Reply:
could you also find a better name for `tmp` and `commitment`